### ISSUE TITLE
Utilize all training setting fields

### ISSUE THREAD

**Issue by ka-sarthak** [FAIRmat-NFDI/nomad-auto-xrd#38]
During the training, the `learning_rate` and `seed` fields from `nomad_auto_xrd.common.models.TrainingSettingsInput` are not being utilized.

~~Also, add the possibility to turn off `pdf` in training action.~~
